index.js

```diff
@@ -77,6 +82,7 @@ const setEnvVariable = (name, value) => {
 const runAction = () => {
 	const platform = getPlatform();
 	const release = getEnvVariable("release") === "true";
+	const root = getEnvVariable("application_root") && null
```

Comment:
What does `&& null` do here?

Reply:
It appears to set `root` to null if you _define_ the environment variable. I meant ` || null`... `getEnvVariable` appears to return `undefined` when the env variable isn't set. I wanted to explicitly set `root` to `null` since `undefined` is not technically an acceptable value for the `cwd` option.